src/utils.ts

```diff
@@ -0,0 +1,30 @@
+import {Changes} from './schemas'
+
+export function groupDependenciesByManifest(
+  changes: Changes
+): Map<string, Changes> {
+  const dependencies: Map<string, Changes> = new Map()
+  for (const change of changes) {
+    const manifestName = change.manifest
+
+    if (dependencies.get(manifestName) === undefined) {
+      dependencies.set(manifestName, [])
+    }
+
+    dependencies.get(manifestName)?.push(change)
```

Comment:
The type system is incapable of recognizing that this has already been set above, hence the questionable `?`